conanfile.py

```diff
@@ -14,8 +14,8 @@ class TaraxaConan(ConanFile):
 
     def requirements(self):
         self.requires("boost/1.76.0")
-        self.requires("cppcheck/2.3")
-        self.requires("openssl/1.1.1k")
+        self.requires("cppcheck/2.5") # 2.6 Segmentation fault (core dumped) while building
+        self.requires("openssl/1.1.1l")
```

Comment:
is there any need for that to be also updated?

Reply:
for openssl not, but why not if it is not breaking anything